requirements/frozen_docs.py2.txt

```diff
@@ -41,8 +41,10 @@ funcsigs==1.0.2
 functools32==3.2.3.post2
 future==0.16.0
 futures==3.0.5
+gevent==1.3.5
```

Comment:
gevent is included in the mocked modules for the docs build, so it doesn't actually need to be installed.